### ISSUE TITLE
highlighting different keywords(types) with different color!

### ISSUE THREAD

**Issue by nish5uec** [react-pdf-viewer/react-pdf-viewer#449]
Hi @phuoc-ng 

I see you are using css element to define the color used while highlighting a text found during a search in pdf.
The requirement here is to highlight different type of keywords with different color if possible.
Not sure about internal implementation, but from client side, i can pass color with the keyword in api and probably you can use it to highlight, otherwise default color if not present.
Let me know if its possible.

Thanks

**Comment by phuoc-ng** [comment]
@nish5uec It was implemented and will be available in the next 2.4.0.

_Demo_

We will add `outline` and `backgroundColor` styles for the highlighting element if the keyword is `document`:

```js
const searchPluginInstance = searchPlugin({
    onHighlightKeyword: (props: OnHighlightKeyword) => {
        if (props.keyword.source === 'document') {
            props.highlightEle.style.outline = '2px dashed blue';
            props.highlightEle.style.backgroundColor = 'rgba(0, 0, 0, .1)';
        }
    },
});
```

<img width="684" alt="Screen Shot 2021-02-02 at 22 32 06" src="https://user-images.githubusercontent.com/57786711/106623433-2fc02f00-65a7-11eb-90d6-f78ed91da12b.png">

**Comment by phuoc-ng** [comment]
Done.
